spec/factories/levels.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :level do
+    name { Faker::Lorem.word }
+    description { Faker::Lorem.word }
+    rubric_criteria
```

Comment:
Be careful with spelling here. You want the singular form, `rubric_criterion`, for the association.